src/group/controller/item.ts

```diff
@@ -97,7 +97,18 @@ class GroupController extends Controller {
     }
 
     if (ctx.request.accepts('text/html')) {
-      ctx.redirect(303, group.href);
+      const group = await principalService.findByExternalId(ctx.params.id, 'group');
```

Comment:
You already have the data for `group` from earlier in the function

Reply:
Fixed :)